ext/LegendDataManagementSolidStateDetectorsExt.jl

```diff
@@ -634,17 +634,25 @@ function create_SSD_config_dict_from_LEGEND_metadata(meta::PropDict, xtal_meta::
     
     slice = Symbol(meta.name[end])
     config_dict["detectors"][1]["semiconductor"]["impurity_density"] = if hasproperty(xtal_meta,:impurity_curve) && hasproperty(xtal_meta.slices, slice)
+        impurity_scale =  hasproperty(xtal_meta.impurity_curve.corrections, :scale) ? xtal_meta.impurity_curve.corrections.scale : 1.0
+        impurity_offset = hasproperty(xtal_meta.impurity_curve.corrections, :offset) ? xtal_meta.impurity_curve.corrections.offset * -1e6 : 0.0 ## 1e9cm^-3 -> mm^-3
```

Comment:
Why minus? `-1e6`. To have it ALWAYS positive, when talking about p-type detectors?

Reply:
Can this be simplified to something like (test if it works):
```suggestion
        impurity_offset = get(xtal_meta.impurity_curve.corrections, :offset, 0) * -1e6
```

Reply:
> Why minus? `-1e6`. To use positive values when talking about p-type detectors? What if one were to try to make it "more n-type"?

Because everything in the metadata is positive. Thus a positive value in `xtal_meta.impurity_curve.corrections.offset` will result in more p-type impurities, and a negative value will make it more n-type.